lms/djangoapps/discussion/rest_api/urls.py

```diff
@@ -31,6 +32,11 @@
         CourseDiscussionSettingsAPIView.as_view(),
         name="discussion_course_settings",
     ),
+    url(
+        fr"^v1/courses/{settings.COURSE_ID_PATTERN}/(?P<thread_key>[a-z0-9]+)/upload$",
```

Comment:
I think it would be great if we can keep the URL `v1/courses/{settings.COURSE_ID_PATTERN}` without the theard key. I think this is the well recognized sytanx around the urls.py.